server.js

```diff
@@ -46,7 +46,7 @@ function create (app) {
 
 require('./lib/bootevent')(env).boot(function booted (ctx) {
     var app = require('./app')(env, ctx);
-    var server = create(app).listen(PORT);
+    var server = create(app).listen(PORT, '::');
```

Comment:
Yes, the behavior without it was broken for IPv6, so it seemed a worthwhile change.  I'm happy to have it reverted while awaiting more testing if needed though.